docs/cep/query-guide/trigger.md

```diff
@@ -3,50 +3,52 @@ sidebar_position: 70
 title: Trigger
 ---
 
-Triggers allow events to be periodically generated. **Trigger definition** can be used to define a trigger. A trigger also works like a stream with a predefined schema.
+Triggers allow events to be periodically generated. A trigger also works like a [stream](stream.md) with a predefined schema.
 
-**Purpose**
+## Purpose
 
-For some use cases the system should be able to periodically generate events based on a specified time interval to perform some periodic executions.
+For some use cases, the system should be able to periodically generate events based on a specified time interval to perform some periodic executions.
 
 A trigger can be performed for a `'start'` operation, for a given `<time interval>`, or for a given `'<cron expression>'`.
 
-**Syntax**
+## Syntax
 
-The syntax for a trigger definition is as follows.
+The syntax for a trigger definition is as follows:
 
-```
+```js
 CREATE TRIGGER <trigger name> at ('start'| every <time interval>| '<cron expression>');
 ```
 
 Similar to streams, triggers can be used as inputs. They adhere to the following stream definition and produce the `triggered_time` attribute of the `long` type.
 
-```
+```js
 CREATE STREAM <trigger name> (triggered_time long);
 ```
 
-The following types of triggeres are currently supported:
+## Supported Triggers
+
+The following types of triggers are currently supported:
 
 |Trigger type| Description|
 |-------------|-----------|
-|`'start'`| An event is triggered when Stream is started.|
+|`'start'`| An event is triggered when a stream is started.|
 |`every <time interval>`| An event is triggered periodically at the given time interval.
 |`'<cron expression>'`| An event is triggered periodically based on the given cron expression. For configuration details, see <a target="_blank" href="http://www.quartz-scheduler.org/documentation/quartz-2.1.7/tutorials/tutorial-lesson-06.html">quartz-scheduler</a>.
```

Comment:
@StoyanTc and @mikeridley, should we have a link to quartz-scheduler here?

Reply:
@diana-macrometa - This link explains the cron expression one could use here. So, it's worth to keep it.